src/aspire/volume/volume.py

```diff
@@ -176,6 +179,51 @@ def _set_symmetry_group(self, value):
             )
         self._symmetry_group = value
 
+    def _symmetry_group_warning(self):
+        """
+        Warn when an operation has the potential to change the symmetry
+        of a volume or the alignment of symmetry axes.
+        """
+        msg = (
+            f"`symmetry_group` attribute is being set to `C1`. This operation may"
+            f" effect the symmetry (or symmetric alignment) of the volume. To reset the"
+            f" symmetry group run `self._set_symmetry_group('{self.symmetry_group}')`."
+        )
+        warnings.warn(msg, UserWarning, stacklevel=4)
+
+    def _result_symmetry(self, other=None):
+        """
+        Check if `other` will alter symmetry of `self` and return resulting symmetry_group.
+
+        :return: SymmetryGroup instance.
+        """
+        # No need to check for C1.
+        if str(self.symmetry_group) == "C1":
```

Comment:
Why not check if it is IdentitySymmetryGroup directly?

Reply:
oof, yeah. done.